### PR TITLE
Feature/#327 faf policy server

### PR DESCRIPTION
**Pull request by micheljung**

### CODE REVIEW

File: server/lobbyconnection.py [hunk 1]
```diff
@@ -628,12 +556,12 @@ def check_version(self, message):
                                      "player_id": player_id
                                  })
 
-            if not self.player_service.is_uniqueid_exempt(player_id):
-                # UniqueID check was rejected (too many accounts or tamper-evident madness)
-                uniqueid_pass = await self.validate_unique_id(cursor, player_id, steamid, message['unique_id'])
-                if not uniqueid_pass:
+            if not self.player_service.is_uniqueid_exempt(player_id) or steamid is not None:
```
Comment (Brutus5000) [inline server/lobbyconnection.py]: Check conformity if steamid is not None?
I'd expect `and steamid is None`

File: server/lobbyconnection.py [hunk 2]
```diff
@@ -612,6 +506,40 @@ def check_version(self, message):
...
+        if response.get('result', '') == 'vm':
+            self.sendJSON(dict(command="notice", style="error",
+                               text="You need to link your account to Steam in order to use FAF in a Virtual Machine. "
+                                    "You can contact an admin on the forums."))
+
+        if response.get('result', '') == 'already_associated':
+            self._logger.warning("UID hit: %d: %s", player_id, uid_hash)
+            self.send_warning("Your computer is already associated with another FAF account.<br><br>In order to "
+                              "log in with a new account, you have to link it to Steam: <a href='" +
+                              config.APP_URL + "/faf/steam.php'>" +
```
Comment (Brutus5000) [inline server/lobbyconnection.py]: This url no longer exists.

File: server/lobbyconnection.py [hunk 3]
```diff
@@ -612,6 +506,40 @@ def check_version(self, message):
                 return False
         return True
 
+    async def check_policy_conformity(self, player_id, uid_hash):
+        request_data = urllib.parse.urlencode(dict(player_id=player_id, uid_hash=uid_hash)).encode()
+        with closing(urllib.request.urlopen(FAF_POLICY_SERVER_BASE_URL + '/verify', request_data)) as response:
+            response = json.loads(response.read().decode())
+
+        if response.get('result', '') == 'vm':
+            self.sendJSON(dict(command="notice", style="error",
```
Comment (Brutus5000) [inline server/lobbyconnection.py]: We should at least add debug logging here.

File: server/lobbyconnection.py (hunk 1 above)
Comment (micheljung) [reply server/lobbyconnection.py]: fixed

File: server/lobbyconnection.py (hunk 2 above)
Comment (micheljung) [reply server/lobbyconnection.py]: Will be fixed in #318

**Pull request merged**